markdown-to-confluence/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "markdown-to-confluence",
```

Comment:
I think we should create a pondevelopment package for this (`@pondevelopment/markdown-to-confluence`)